annsa/tests/test_training_dnn.py

```diff
@@ -76,8 +87,10 @@ def test_dnn_training():
         earlystop_patience=0,
         verbose=1,
         print_errors=0,
-        max_time=3600,
         obj_cost=model.cross_entropy,
         earlystop_cost_fn=model.f1_error,
         data_augmentation=model.default_data_augmentation,)
     pass
```

Comment:
This test doesn't test anything.

Reply:
@kamuda1

Reply:
yes, but that's not a test.

Reply:
tests are intended to ensure the code runs _correctly_ . Exceptions are already present to throw an error if a function won't even run.